app/uk/gov/hmrc/pegaproofofconceptstubs/controllers/PegaController.scala

```diff
@@ -33,19 +34,27 @@ class PegaController @Inject() (cc: ControllerComponents)()
   val startCase: Action[StartCaseRequest] = Action(parse.json[StartCaseRequest]) { implicit request =>
     logger.info("[OPS-11581] payload submitted with value: " + request.body.toString)
     logger.info("[OPS-11785] responding with: " + StartCaseResponse.defaultResponse.toString)
+    logger.info("Received access token: " + request.headers.get(HeaderNames.AUTHORIZATION).toString)
```

Comment:
could we perhaps combine the three logs above into one and the two logs in `getCase` into one? It'd make it a bit easier to read the logs, or at least it'd be more obvious that the logs corresponded with the same transaction

Reply:
Sure. Will do.